src/marqo/index.py

```diff
@@ -460,7 +459,8 @@ def verbosely_add_docs(i, docs):
             errors_detected = False
 
             t0 = timer()
-            res = self.http.post(path=path_with_query_str, body=docs, index_name=self.index_name,)
+            body = {"documents": docs, **base_body}
+            res = self.http.post(path=path_with_query_str, body=body)
```

Comment:
add `index_name` so that Cloud v2 requests go directly to index

Reply:
thanks!